### PR TITLE
[Hub Generated] Review request for Microsoft.CostManagement to add version preview/2019-03-01-preview

### PR DESCRIPTION
**Pull request by taleib**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/taleib/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2019-03-01-preview...master).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by taleib** [comment]
@noferrar

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Can you please remove the BOM from ExternalBillingAccountForecast.json

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json [hunk 1]
```diff
@@ -1146,6 +1146,57 @@
         }
       }
     },
+    "/providers/Microsoft.CostManagement/externalBillingAccounts/{externalBillingAccountName}/Forecast": {
```
Comment (majastrz) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json]: >Forecast [](start = 94, length = 8)

The ARM resource provider contract requires plural camelCased resource names ([https://github.com/Azure/azure-resource-manager-rpc](https://github.com/Azure/azure-resource-manager-rpc)). Please rename to `forecasts`.

**Review by majastrz — APPROVED** [review]
Looks good from ARM side.

**Comment by taleib** [comment]
> Can you please remove the BOM from ExternalBillingAccountForecast.json

resolved

**Comment by jhendrixMSFT** [comment]
There are several model validation failures due to invalid examples that need to be addressed.  Please see the following log.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=30021

**Comment by taleib** [comment]
> There are several model validation failures due to invalid examples that need to be addressed. Please see the following log.
> https://dev.azure.com/azure-sdk/public/_build/results?buildId=30021

these errors are from another service and not Forecast. can we advance without addressing them?

**Comment by jhendrixMSFT** [comment]
@taleib these errors cannot be suppressed.  Please file an issue to track fixing them, then we can go ahead with the PR.

**Comment by taleib** [comment]
> @taleib these errors cannot be suppressed. Please file an issue to track fixing them, then we can go ahead with the PR.

resolved

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**